<?xml version="1.0"?>

<launch>
	<arg name="indoor" default="true"/>

	<param name="indoor" type="boolean" value="$(arg indoor)" />

	<param name="drone_ip" type="string" value="192.168.53.1" /> <!-- This is the skycontroller IP -->

	<node name="anafi_bridge" pkg="olympe_bridge" type="anafi_bridge.py" output="screen"/>
</launch>
